src/stories/Button.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import { ComponentMeta, ComponentStory } from "@storybook/react";
```

Comment:
Do you anticipate deleting the `stories/` directory once we start development?

Reply:
Yep, I will delete the entire directory! In my Emotion/LG PR, I will add a story for a custom component using Emotion and LG and make sure that everything still works